indico/web/client/js/react/components/principals/Search.jsx

```diff
@@ -102,9 +102,10 @@ const searchFactory = config => {
     );
   };
 
-  const SearchForm = ({onSearch, favorites, isAdded, onAdd, onRemove, single}) => {
+  const SearchForm = ({onSearch, favoritesController, isAdded, onAdd, onRemove, single}) => {
     const initialFormValues = useContext(InitialFormValuesContext);
     const [dropdownOpen, setDropdownOpen] = useState(false);
+    const favorites = favoritesController ? favoritesController[0] : [];
```

Comment:
If you wanna be fancy, but either is fine:
```suggestion
    const favorites = favoritesController?.[0] ?? [];
```